nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment-api/src/main/java/org/nuxeo/ecm/platform/comment/api/AnnotationConstants.java

```diff
@@ -34,14 +34,17 @@ private AnnotationConstants() {
 
     public static final String ANNOTATION_DOC_TYPE = "Annotation";
 
+    /** @since 11.1 */
+    public static final String ANNOTATION_SCHEMA = "annotation";
+
```

Comment:
just realized it was added in the last commit instead of rework constants location

Reply:
Will fix it, thanks.

Reply:
In fact this is not a rework, the annotation schema wasn't defined previously, it was introduced by these changes.

Reply:
Okay ! Got confused as the comment came first !